core/eolearn/tests/test_eodata.py

```diff
@@ -178,6 +178,23 @@ def test_delete_existing_feature(feature: FeatureSpec, mini_eopatch: EOPatch) ->
             assert_array_equal(old[old_feature], mini_eopatch[old_feature])
 
 
+@pytest.mark.parametrize("feature_type", [FeatureType.DATA, FeatureType.TIMESTAMPS])
+def test_delete_existing_feature_type(feature_type: FeatureType, mini_eopatch: EOPatch) -> None:
+    old = mini_eopatch.copy(deep=True)
+
+    del mini_eopatch[feature_type]
+    assert feature_type not in mini_eopatch
+
+    for ftype, fname in old.get_features():
+        if ftype != feature_type:
+            assert_array_equal(old[ftype, fname], mini_eopatch[ftype, fname])
```

Comment:
i think we should not use `assert_array_equal` on BBOX

Reply:
it cant be :) because BBOX cannot be deleted